runtime/criusupport/criusupport.cpp

```diff
@@ -1052,7 +1058,7 @@ Java_org_eclipse_openj9_criu_CRIUSupport_checkpointJVMImpl(JNIEnv *env,
 wakeJavaThreadsWithExclusiveVMAccess:
 
 		vm->extendedRuntimeFlags2 &= ~J9_EXTENDED_RUNTIME2_CRIU_SINGLE_THREAD_MODE;
-
+		vm->extendedRuntimeFlags2 &= ~J9_EXTENDED_RUNTIME2_CRIU_SINGLE_THROW_BLOCKING_EXCEPTIONS;
```

Comment:
Probably not an issue, should this be before `vm->extendedRuntimeFlags2 &= ~J9_EXTENDED_RUNTIME2_CRIU_SINGLE_THREAD_MODE;`?

Reply:
Should be combined with the line below (for compactness, not correctness).